sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayout.java

```diff
@@ -94,45 +101,71 @@ public int getViewHeight() {
   }
 
   public boolean isFullscreen() {
-    return isFullscreen;
+    return fullscreen;
```

Comment:
this method doesn't receive a parameter, or am i understanding the question incorrectly?